test/emailpassword/config.test.js

```diff
@@ -42,4 +44,195 @@ describe(`configTest: ${printPath("[test/config.test.js]")}`, function () {
         await killAllST();
         await cleanST();
     });
+
+    //  TODO: test config for emailpassword module
+    it("test default config for emailpassword module", async function () {
+        await startST();
+        STExpress.init({
+            supertokens: {
+                connectionURI: "http://localhost:8080",
+            },
+            appInfo: {
+                apiDomain: "api.supertokens.io",
+                appName: "SuperTokens",
+                websiteDomain: "supertokens.io",
+            },
+            recipeList: [EmailPassword.init()],
+        });
+
+        let emailpassword = await EmailPasswordRecipe.getInstanceOrThrowError();
+        assert(emailpassword.config.signUpFeature.formFields.length === 2);
```

Comment:
I'm sure there are more checks that you can do here. This is incomplete checking.